server/tests/controller/signup.spec.js

```diff
@@ -0,0 +1,149 @@
+import chai from "chai";
+import uniqid from "uniqid";
+import chaiHttp from "chai-http";
+import app from "../../index";
+import database from "../../models/index";
+import userData from "../fixtures/models/userData";
+import { generateJWT, decodeJWT, getJWTConfigs } from "../../helpers/jwt";
+import { getBaseUrl } from "../../helpers/userHelpers";
+
+chai.use(chaiHttp);
+const { expect, assert } = chai;
+
+beforeEach(async () => {
+  await sequelize.sync({ force: true }).catch(() => {});
+});
+const userDependencies = async () => {
+  const user = await User.create({
+    firstName,
+    lastName,
+    password,
+    email: "solo@gmail.com"
+  });
+  const userId = user.get("id");
+
+  return Promise.resolve({ userId, email: user.get("email") });
```

Comment:
The variables are not descriptive enough I don't understand what is happening here